ggshield/output/gitlab_webui/gitlab_webui_output_handler.py

```diff
@@ -0,0 +1,64 @@
+from pygitguardian.models import PolicyBreak
+
+from ggshield.filter import censor_match
+from ggshield.output.output_handler import OutputHandler
+from ggshield.scan import ScanCollection
+from ggshield.text_utils import pluralize, translate_validity
+
+
+def format_policy_break(policy_break: PolicyBreak) -> str:
+    """Returns a string with the policy name, validity and a comma-separated,
+    double-quoted, censored version of all `policy_break` matches.
+
+    Looks like this:
+
+    PayPal OAuth2 Keys (Validity: Valid, id="aa*******bb", secret="cc******dd")
+    """
+    match_str = ", ".join(
+        f'{x.match_type}: "{censor_match(x)}"' for x in policy_break.matches
+    )
+    validity = translate_validity(policy_break.validity)
+    return f"{policy_break.break_type} (Validity: {validity}, {match_str})"
+
+
+class GitLabWebUIOutputHandler(OutputHandler):
+    """
+    Terse OutputHandler optimized for GitLab Web UI, because GitLab Web UI only shows
+    lines starting with GL-HOOK-ERR.
+
+    See https://docs.gitlab.com/ee/administration/server_hooks.html#custom-error-messages
+    """
+
+    def __init__(self) -> None:
+        super().__init__(show_secrets=False, verbose=False)
```

Comment:
`show-secret` parameter default value is None. Any reason I miss we don't want to show secrets if our users explicitly set it to True?

Reply:
Good catch, fixed this.